crates/vm/src/system/memory/online.rs

```diff
@@ -373,13 +373,11 @@ impl GuestMemory {
 #[repr(C)]
 #[derive(Clone, Copy, PartialEq, Eq, Debug, Default, derive_new::new)]
 pub struct AccessMetadata {
-    /// The starting pointer of the access
-    pub start_ptr: u32,
-    /// The block size of the memory access
-    pub block_size: u32,
-    /// The timestamp of the last access.
-    /// We don't _have_ to store it, but this is probably faster
-    /// in terms of cache locality
+    /// Offset to block start (in ALIGN units). If 0, this IS the block start.
+    pub offset_to_start: u8,
```

Comment:
nit: to follow Xinding's principles, smaller sized items should be at the bottom (here it makes no difference, sizeof is still 8)

Reply:
makes sense